src/styles/blocks/box.scss

```diff
@@ -0,0 +1,25 @@
+.box {
+  display: grid;
+  grid-template-columns: repeat(3, minmax($box-size, 1fr));
+
+  &__item {
+    display: flex;
```

Comment:
You could use `display: grid` for this purpose as well

Reply:
changed